### ISSUE TITLE
https://examples.yetanother.blog - page not found error

### ISSUE THREAD

**Issue by coliff** [yetanother-blog/examples#5]
The URL for this project is https://examples.yetanother.blog/ - but this gives a 404 page not found error.

**Comment by andrew-secret** [comment]
@coliff you're right. Maybe we could add some meaningful content there. At the moment we always serve the example in a separate folder e.g. [https://examples.yetanother.blog/dark-mode-toggle](https://examples.yetanother.blog/dark-mode-toggle)

Basic idea was to:
- create a folder (e.g `my-awesome-example`)
- paste `index.html`, `styles.css` and `main.js` into that folder
- then you can serve this example via `https://examples.yetanother.blog/my-awesome-example`

Thanks for your feedback 🙏

**Comment by HenrikFricke** [comment]
Actually interesting if we could build some tooling around our examples so we would have a simple landing page with a list of examples. maybe even connect them with the blog posts 🤯
